docs/kind.md

```diff
@@ -24,6 +24,9 @@ To create a two worker Node cluster with Antrea installed using scripts, do
 ./ci/kind/kind-setup.sh create CLUSTER_NAME
 ```
 
+This command will execute `kubectl` commands to set up the cluster, and requires
+that `kubectl` is present in your `PATH`.
```

Comment:
s/is present/be present

Reply:
You like to change to "requires `kubectl` to be present"?

Reply:
no, "requires that `kubectl` be present"

Reply:
Why? My grammar told me "that kubectl is present" is an object clause, and should use regular tense.

Reply:
The clause should be in the subjunctive: https://ell.stackexchange.com/questions/46831/require-that-x-be-used-vs-require-x-to-be-used

Reply:
Here present is adjective not verb?

Reply:
I searched with google. Seems there are indeed more results of "be present" than "is present". So, you must be right. Updated it.
Good to learn new grammar!